doc-first-package.md

```diff
@@ -0,0 +1,56 @@
+# My first package walk-through
+You're missing a good analysis tools in this package repository?
+See below on how you can create the package via a GitHub issue or manually with the support of a package template script.
+
+## Automatic pull request via new issue
+* go to Issues - New Issue
+* select the appropriate package type
+  * for a "METAPACKAGE" select the respective option, all other packages select "NEW PACKAGE"
+* fill out the required fields
+* submit your new issue
+* a maintainer will review your submission and label it for automatic package creation
+  * the automated processing includes linting and testing
+* if changes are required, you can
+  * download the created branch and modify it locally
```

Comment:
what about linking Github documentation about how to do it? https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally

I think we should add that they may need to open a new PR because I think only maintainers can push to a PR pen by someone else.